### ISSUE TITLE
PDF form of the manual for 6.0-BETA* is not available

### ISSUE THREAD

**Issue by Idloj** [NetLogo/NetLogo#1257]
Going to the docs of the betas and clicking "printable form" to get a PDF results in a 404

**Comment by mrerrormessage** [comment]
Yep, thanks for reporting!

**Comment by Idloj** [comment]
This might be related to an error I get when trying to run `htmlDocs` or `manualPDF` in sbt:
```
[trace] Stack trace suppressed: run last netlogo/*:manualPDF for the full output.
[error] (netlogo/*:manualPDF) com.typesafe.config.ConfigException$Missing: No configuration setting found for key 'markdownTemplate'
[error] Total time: 6 s, completed Dec 20, 2016 10:51:37 PM
```

The error is in the usage of HoconParser by generateHTMLPageForExtension. Stack trace:
```
com.typesafe.config.ConfigException$Missing: No configuration setting found for key 'markdownTemplate'
	at com.typesafe.config.impl.SimpleConfig.findKeyOrNull(SimpleConfig.java:152)
	at com.typesafe.config.impl.SimpleConfig.findOrNull(SimpleConfig.java:170)
	at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:184)
	at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:189)
	at com.typesafe.config.impl.SimpleConfig.getString(SimpleConfig.java:246)
	at org.nlogo.build.HoconParser$.parseConfig(HoconParser.scala:198)
	at NetLogoDocs.generateHTMLPageForExtension(NetLogoDocs.scala:94)
	at NetLogoDocs$$anonfun$generateExtensionDocs$1.apply(NetLogoDocs.scala:136)
	at NetLogoDocs$$anonfun$generateExtensionDocs$1.apply(NetLogoDocs.scala:133)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at NetLogoDocs.generateExtensionDocs(NetLogoDocs.scala:133)
	at NetLogoDocs.generateDocs(NetLogoDocs.scala:125)
	at NetLogoDocs.generatePDF(NetLogoDocs.scala:60)
	at Docs$$anonfun$settings$8.apply(Docs.scala:50)
	at Docs$$anonfun$settings$8.apply(Docs.scala:49)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by mrerrormessage** [comment]
The manual was just not included in the webpage. Fixed in 6.0. Thanks for reporting!
